New blog: Implementing Open-Source AI Models

AnthonyCvn opened the pull request:
New blog post: "From Lab to Live: Implementing Open-Source AI Models for Real-time Unsupervised Anomaly Detection in Images"

AnthonyCvn commented:
@atimin I can't assign you as reviewer. Can you add me as collaborator for this repo?

atimin commented on _posts/2023-10-13-Implementing-open-source-ai-anomaly-detection.md:
```diff
@@ -0,0 +1,105 @@
+
+Once satisfied with its performance and reliability, you can then switch over from your old system to this newly deployed AI model.
+
+### Harnessing ReductStore: Storing AI Tags and Models at the Edge with a Time-Series Database for Blob Data
```
Is the 'label' term more common than `tag` in the context of AI?

AnthonyCvn replied:
Both tag and label are commonly used

atimin replied:
But anyway, let's use label because it's part of the ReductStore vocabulary, like "Data Labeling & Filtering" in our landing page. Just to show that we mean the same thing.

AnthonyCvn replied:
Done

atimin commented on _posts/2023-10-13-Implementing-open-source-ai-anomaly-detection.md:
```diff
@@ -0,0 +1,105 @@
+- AI
+---
+
+![Photo by Randy Fath](https://storage.googleapis.com/reductstore_blog_images/2023-10-13/randy-fath-chess.jpeg "Photo by Randy Fath")
```
pelase, store the image in /assets directory.

AnthonyCvn replied:
I'll give it a second try.

AnthonyCvn replied:
I also made the change for past blogs in a separate PR  🤓

AnthonyCvn replied:
I used assets/blog directory for all blog related assets sorted by dates. I would use the img directory for images related to the website.

atimin commented on _posts/2023-10-13-Implementing-open-source-ai-anomaly-detection.md:
```diff
@@ -0,0 +1,105 @@
+
+To better visualize how ReductStore can integrate with your machine learning workflow, from data capture to inference, consider the following diagram. It provides an overview of how we can make the most out of AI tags and models stationed at the edge.
+
+![ML Data Flow Diagram](https://storage.googleapis.com/reductstore_blog_images/2023-10-13/ml_data_flow_diagram.jpeg)
```
I would invest a little bit more time into the diagram. It is correct but  I would like to add some details. If you want I could draw as I see it. 

PS: it should be in /assets and the source draw.io file as well.

AnthonyCvn replied:
I agree, I'm already working on a better version for the Whitepaper.
Let me share this one in /assets folder so you have access to it as well.

AnthonyCvn commented:
@atimin I updated the diagram, added more information about how to use Anomalib, and changed some phrasing and sentences. I'm more satisfied with the result :)

AnthonyCvn commented:
@atimin I updated the diagram, added more information about how to use Anomalib, and changed some phrasing and sentences. I'm more satisfied with the result :)
